wafer/talks/forms.py

```diff
@@ -20,6 +20,14 @@ def get_talk_form_class():
 class TalkForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user')
+        initial = kwargs.setdefault('initial', {})
+        authors = initial.setdefault('authors', [self.user])
+
+        if not (settings.WAFER_PUBLIC_ATTENDEE_LIST
+                or self.user.has_perm('talks.change_talk')):
+            self.base_fields['authors'].limit_choices_to = {
+                'id__in': [author.id for author in authors]}
```

Comment:
Just confirming that ordinary users then cannot add anyone else to their talk? I think that's okay, just checking that other agree.